leshan-server-demo/src/main/java/org/eclipse/leshan/server/demo/servlet/json/JacksonLwM2mNodeSerializer.java

```diff
@@ -85,6 +85,7 @@ public void serialize(LwM2mNode src, JsonGenerator gen, SerializerProvider provi
 
         }
 
+        gen.enable(JsonGenerator.Feature.WRITE_BIGDECIMAL_AS_PLAIN);
```

Comment:
I guess this one is not expected ?

Reply:
Sorry, there were a lot of those serializer classes so I must have missed one. I'll reorganize the commits so we have 3 instead of 10 once everything's fine.